daal4py/sklearn/ensemble/_forest.py

```diff
@@ -326,7 +342,7 @@ def _daal_fit_regressor(self, X, y, sample_weight=None):
     # create algorithm
     dfr_algorithm = daal4py.decision_forest_regression_training(
         fptype=getFPType(X),
-        method='defaultDense',
+        method='hist',

```

Comment:
there is no hist in the old daal. We must support the previous version (checked in public CI)

Reply:
correct version for 'hist' was added, examples were added